lib/modules/easterEgg.js

```diff
@@ -23,11 +21,59 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
-	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
+	});
 };
 
 export function konamiActive() {
 	return konami && konami.almostThere;
 }
+
+/*
+ * Konami-JS ~
+ * :: Now with support for touch events and multiple instances for
+ * :: those situations that call for multiple easter eggs!
+ * Code: https://github.com/snaptortoise/konami-js
+ * Examples: http://www.snaptortoise.com/konami-js
+ * Copyright (c) 2009 George Mandis (georgemandis.com, snaptortoise.com)
+ * Version: 1.4.6 (3/2/2016)
+ * Licensed under the MIT License (http://opensource.org/licenses/MIT)
+ * Tested in: Safari 4+, Google Chrome 4+, Firefox 3+, IE7+, Mobile Safari 2.2.1 and Dolphin Browser
+ */
+function Konami(callback) {
+	const konami = {
+		input: '',
+		pattern: '38384040373937396665',
+		almostThere: false,
+		prepattern: '383840403739373966',
+		load: () => {
+			document.addEventListener('keydown', event => {
+				konami.input += event ? event.keyCode : '';
```

Comment:
Nit: we can also remove the null-check from this, i.e. just `konami.input += event.keyCode;`